atomic_reactor/plugins/post_export_operator_manifests.py

```diff
@@ -117,6 +117,8 @@ def run(self):
         tar_archive.extractall(manifests_archive_dir)
         manifests_path = os.path.join(manifests_archive_dir, MANIFESTS_DIR_NAME)
 
+        self._verify_csv(manifests_path)
```

Comment:
This verification should run only for operator bundles not for appregistry manifests

Reply:
@MartinBasti Any hint how to determine current built image is an operator bundle not appregistry manifests? Thanks.

Reply:
https://github.com/containerbuildsystem/atomic-reactor/blob/2ab5516cf569015ad794c709ea9869a40a61dabd/atomic_reactor/util.py#L1542